src/AbstractSQLRules2SQL.ts

```diff
@@ -996,6 +1030,14 @@ const typeRules: Dictionary<MatchFn> = {
 			return `TIME(${date})`;
 		}
 	},
+	ToJSON: (args, indent) => {
+		checkMinArgs('ToJSON', args, 1);
+		if (engine !== Engines.postgres) {
+			throw new SyntaxError('ToJSON not supported on: ' + engine);
+		}
+		const column = AnyValue(getAbstractSqlQuery(args, 0), indent);
```

Comment:
This doesn't need to be a column specifically so it's better to use a more generic name to avoid confusion:
```suggestion
		const value = AnyValue(getAbstractSqlQuery(args, 0), indent);
```